backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -63,11 +65,27 @@ public void testCreateShop() throws Exception {
         assertEquals("Shop Phone Number ", PHONE_NUMBER, inMemoryVirtualQueueRepository.getShopMap().get(shopId).getPhoneNumber());
         assertEquals("Shop Type ", SHOP_TYPE, inMemoryVirtualQueueRepository.getShopMap().get(shopId).getShopType());
     }
-    
+
     @Test
     public void testGetWaitingTime() throws Exception {
         long waitingTime = inMemoryVirtualQueueRepository.getWaitingTime();
-        
+
         assertEquals("Waiting Time per customer is", WAITING_TIME_MINS, waitingTime);
     }
+
+    @Test
```

Comment:
Follow the AAA pattern
https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Reply:
Changed.